src/tools/stereoCalib/include/stereoCalibModule.h

```diff
@@ -21,10 +21,11 @@ class stereoCalibModule:public RFModule
     string outputPortNameLeft;  
     string handlerPortName;
     string outputCalibPath;
+	string dir;
```

Comment:
Hi @lsrosa , thanks for the PR. In general it is a good practice to not mix tabs and spaces, as the appearance of tabs changes from system to system. Depending on your editor, you can configure it to emit spaces whenever you press the tab key on the keyboard, feel free to ping me if you are not able to find how to modify this setting in the configuration of your editor.